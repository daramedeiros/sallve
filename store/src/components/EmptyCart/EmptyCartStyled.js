import styled from 'styled-components';

export const EmptyCartComponent = styled.div`
    display:block;
    alien-items:center;
    width: 100%;
    text-align:center;

    img {
        padding-bottom: 25px;
        width: 100px;
    }
`;
