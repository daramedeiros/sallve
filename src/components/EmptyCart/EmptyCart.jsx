import React from 'react';
import  CarryBag  from '../../assets/img/carry-bag.png';

import { EmptyCartComponent } from './EmptyCartStyled';

const EmptyCart = () => (
  <EmptyCartComponent>
    <img alt="Icone de Sacola de compras" src={CarryBag} />
    <h3>Ops! sua sacola ainda está vazia. vem cá descobrir nossas fórmulas e escolher a sua!</h3>
  </EmptyCartComponent>
)

export default EmptyCart;
