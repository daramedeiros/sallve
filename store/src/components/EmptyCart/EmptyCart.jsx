import React from 'react';
import  CarryBag  from '../../assets/img/carry-bag.png';

import { EmptyCartComponent } from './EmptyCartStyled';

const EmptyCart = () => (
  <EmptyCartComponent>
    <img alt="Icone de Sacola de compras" src={CarryBag} />
    <p>Ops! sua sacola ainda está vazia. vem cá descobrir nossas fórmulas e escolher a sua!</p>
  </EmptyCartComponent>
)

export default EmptyCart;
