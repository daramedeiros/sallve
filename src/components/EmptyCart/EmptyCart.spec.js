import { render } from '@testing-library/react';

import EmptyCart from './index';

describe('Add Button Component', () => {
  describe('basic functionality', () => {
    it('should render with default styles', async () => {
      const { container } = render(<EmptyCart />);

      expect(container).toMatchSnapshot();
    });
  });
});