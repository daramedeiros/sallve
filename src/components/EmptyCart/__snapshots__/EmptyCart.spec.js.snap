// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add Button Component basic functionality should render with default styles 1`] = `
<div>
  <div
    class="sc-bdnxRM fbSxoN"
  >
    <img
      alt="Icone de Sacola de compras"
      src="carry-bag.png"
    />
    <h3>
      Ops! sua sacola ainda está vazia. vem cá descobrir nossas fórmulas e escolher a sua!
    </h3>
  </div>
</div>
`;
